src/plugins/MamicoCoupling.cpp

```diff
@@ -0,0 +1,80 @@
+#include "MamicoCoupling.h"
+#include "Domain.h"
+
+#ifdef MAMICO_COUPLING
+#include <coupling/interface/impl/ls1/LS1MamicoCouplingSwitch.h>
+//#include <LS1MamicoCouplingSwitch.h>
+#endif
+
+void MamicoCoupling::readXML(XMLfileUnits& xmlconfig)
+{	
+    return;
+}
+
+void MamicoCoupling::init(ParticleContainer* particleContainer,
+		DomainDecompBase* domainDecomp, Domain* domain)
+{
+    #ifdef MAMICO_COUPLING
+    if(_macroscopicCellService==NULL) 
+    {
+        _macroscopicCellService = (coupling::services::MacroscopicCellServiceImpl<ls1::LS1RegionWrapper,3>*)
+                coupling::interface::MamicoInterfaceProvider<ls1::LS1RegionWrapper,3>::getInstance().getMacroscopicCellService();
+	}
+    //since using mamico thermostat, switch off ls1 thermostat
+    domain->thermostatOff();
```

Comment:
document in the plugin description that this always disables the ls1 thermostat.